packages/frontend/src/components/ProjectConnection/WarehouseForms/PostgresForm.tsx

```diff
@@ -410,28 +331,37 @@ const PostgresForm: FC<{
                                 <TextInput
                                     label="SSH Remote Host"
                                     disabled={disabled}
-                                    {...register('warehouse.sshTunnelHost')}
+                                    name="warehouse.sshTunnelHost"
+                                    {...form.getInputProps(
+                                        'warehouse.sshTunnelHost',
+                                    )}
                                 />
-                                <Controller
+
+                                <NumberInput
                                     name="warehouse.sshTunnelPort"
-                                    defaultValue={22}
-                                    render={({ field }) => (
-                                        <NumberInput
-                                            {...field}
-                                            label="SSH Remote Port"
-                                            disabled={disabled}
-                                        />
+                                    {...form.getInputProps(
+                                        'warehouse.sshTunnelPort',
                                     )}
+                                    defaultValue={
+                                        PostgresDefaultValues.sshTunnelPort
+                                    }
```

Comment:
how does `defaultValue` prop behave with the controlled input? I thought they had some issues together